.github/workflows/makefile.yml

```diff
@@ -0,0 +1,88 @@
+name: Makefile CI
+
+on:
+  push:
+    branches: [ "trunk" ]
+  pull_request:
+    branches: [ "trunk" ]
```

Comment:
I would suggest using

```
on:
    push
    pull_request:
```

only, so that it can run in custom branches in forks for example. (This way, you would have to use trunk, or modify the CI).

Reply:
I've just noticed that the original repository doesn't even use trunk, so this should be changed.

Reply:
fixed.